draft-pda-core.md

```diff
@@ -166,24 +169,44 @@ limited set of aggregation functions, some of which we highlight below:
 This variety of aggregate types is sufficient to support a wide variety of
 data aggregation tasks.
 
-
-## Validating inputs
-
-A crucial task of any data collection pipeline is ensuring that the input data
-is valid. Going back to the example above, it's often useful to assert that each
-measurement is in a certain range, e.g., [0, 2^k) for some k. This straight-forward
-task is complicated in Prio by the fact that the inputs are secret shared. In
-particular, a malicious client can corrupt the computation by submitting random
-integers instead of a proper secret sharing of a valid input.
-
-To solve this problem, Prio introduces a light-weight zero-knowledge proof
-system designed to operate on secret shared data. In addition to its input
-share, each client sends to each aggregator a share of a "proof" of the input's
-validity. The aggregators use these proof shares in a protocol designed to
-establish the input's validity, without leaking the input itself. We describe
-this input-validation protocol in detail in [[TODO:citeme]].
-
-## Assembling Reports
+**Hits.**
+A common PA task that can't be solved efficiently with Prio is the
+`t`-*heavy-hitters* problem {{BBCp21}}. In this setting, each user is in
+possession of a single `n`-bit string, and the goal is to compute the compute
+the set of strings that occur at least `t` times.
```

Comment:
```With Prio, it would be necessary to encode 2^n values, where all but one is zero. This is clearly inefficient.```

Reply:
Is that true? I believe there's more to it than this.

Reply:
Why?

Reply:
As with the other text, I think the purpose of this section is to give people the right intuition.

What's misleading about what I said?

Reply:
There's two things: 
1. Instead of "encode 2^n values" it would be more accurate to say "generate a proof of length `O(some_function_of(n))`". where `some_function_of(.)` depends on the proof strategy. I suppose we could just say "generate a huge proof".
2. The "collect" step doesn't work the same way. It's not a simple matter of adding up shares; the collector has to interact with the aggregators over several rounds.

Reply:
I think the intuition ("should I use Prio or HH?") is good to add here. (@ekr's suggestion seems to clarify the "this can't be solved efficiently claim" with a simple example, so I'd support adding it)

Reply:
How about: `One reason Prio isn't applicable to the use case is that the proof that is generated would be huge.`

Reply:
Done.